src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/RespondentLiPResponse.java

```diff
@@ -13,4 +18,19 @@ public class RespondentLiPResponse {
 
     private MediationLiP respondent1MediationLiPResponse;
     private FinancialDetailsLiP respondent1LiPFinancialDetails;
+    private String respondent1ResponseLanguage;
+
+    @JsonIgnore
+    public boolean isRespondentResponseBilingual(CaseData caseData) {
+        String responseLanguage = Optional.ofNullable(caseData.getCaseDataLiP())
+            .map(CaseDataLiP::getRespondent1LiPResponse)
+            .map(RespondentLiPResponse::getRespondent1ResponseLanguage)
+            .orElse(null);
+
+        if (responseLanguage != null && Language.BOTH.toString().equals(responseLanguage)) {
```

Comment:
You can just return the result of this condition here (also no need for the responseLanguage null check)
```suggestion
        return Language.BOTH.toString().equals(responseLanguage);
```

Reply:
updated as per Leo's suggestion.